src/main/webapp/css/index.css

```diff
@@ -0,0 +1,6 @@
+#city{
```

Comment:
Sorry if I'm missing something obvious--what is the city ID for? And would it make sense to put this styling into main.css instead of creating a new file?

Reply:
Ahhh I was going to add an image and different styles the to index page but changed my mind and forgot to delete this!